### ISSUE TITLE
Region freezes vvvv

### ISSUE THREAD

**Issue by velcrome** [wirmachenbunt/AutomataUI#8]
So if you start drawing a new region, and happen to extend it outside the bounds of AutomataUI's window, it will freeze vvvv eternally once you release the mouse button

**Comment by velcrome** [comment]
So I guess the reason is the little pop up window asking for the name of the new region. Since this window is outside the screen, it just "seems" to hang.

**Comment by wirmachenbunt** [comment]
exactly, pressing enter should close the window, even if its invisible. i can have a look to make the window always appear somewhere on the desktop or rather stop creating regions larger than the desktop

**Comment by velcrome** [comment]
yes, clamping regions inside the window seem a sensible idea
